models/models.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+import "time"
+
+type User struct {
+	ID        int       `json:"user_id"`
+	Email     string    `json:"email"`
+	Password  string    `json:"_"`
```

Comment:
```suggestion
	Password  string    `json:"-"`
```
jsonとして表示しない場合は-を指定する。ゼロ値を表示したくない場合はomitemptyを利用する

https://dev.classmethod.jp/articles/struct-json/